rules/regions.smk

```diff
@@ -46,18 +41,21 @@ rule clean_export_regions:
         "auspice/ncov*_gisaid.json",
         "auspice/ncov*_zh.json",
         "config/colors*.tsv"
+    conda: "../envs/nextstrain.yaml"
     shell:
         "rm {params}"
 
 # Allows 'normal' run of export to be forced to correct lat-long & ordering
 # Just runs this, not ZH & GISAID, to speed up & reduce errors.
+# Runs an additional script to give a list of locations that need colors and/or lat-longs
 rule export_all_regions:
     input:
-        colors_file = expand("config/colors{region}.tsv", region=REGIONS),
-        auspice_json = expand("results/ncov_with_accessions{region}.json", region=REGIONS),
+        colors_file = expand("config/colors_{region}.tsv", region=REGIONS),
+        auspice_json = expand(REGION_PATH + "ncov_with_accessions.json", region=REGIONS),
         lat_longs = files.lat_longs,

```

Comment:
```suggestion
        lat_longs = config["files"]["lat_longs"]
```
This is giving me an error - the above seems to work - is that right?

Reply:
Ps - Not sure why this is showing on the 'wrong side' of the file.... Hope it works anyway.